client/components/ListOfProjects/ListOfProjects.scss

```diff
@@ -8,6 +8,9 @@
 .title {
   font-size: 20px;
   margin-bottom: 15px;
+  padding: 15px;
+  color: white;
```

Comment:
check out this file: _variables.scss
It contains variables for colours...if #ed4b3f does not exist, create a new variable and use it